pkg/vulnsrc/osv/osv.go

```diff
@@ -169,6 +169,9 @@ func (vs VulnSrc) commit(tx *bolt.Tx, eco ecosystem, entry Entry) error {
 
 					// e.g. {"introduced": "1.2.0}, {"fixed": "1.2.5}
 					vulnerable = fmt.Sprintf("%s, <%s", vulnerable, event.Fixed)
+				case event.LastAffected != "":
+					unaffectedVersions = append(unaffectedVersions, fmt.Sprintf(">%s", event.LastAffected))
```

Comment:
Do we need `unaffectedVersions`? For example, `VulnerableVersions: []string{">=0, <=3.8.4"}` already describes `>3.8.5` is not affected.